kategory-docs/docs/docs/datatypes/try/README.md

```diff
@@ -4,6 +4,182 @@ title: Try
 permalink: /docs/datatypes/try/
 ---
 
-## Try 
+## Try
+
+Kategory has a [lots of different types of error handling and reporting](http://kategory.io/docs/patterns/error_handling/), which can make it difficult to decide which one is best for your situation. 
```

Comment:
Replace: `which can make it difficult to decide which one is best for your situation`
For: `which allows you to choose the best strategy for your situation`

Reply:
https://github.com/kategory/kategory/pull/458/commits/2cb2afcae14684d4cf6a17c52366463cc2607bd6